consensus/service/src/api/client_api_service.rs

```diff
@@ -92,12 +132,28 @@ impl ClientApiService {
         msg: Message,
     ) -> Result<ProposeTxResponse, ConsensusGrpcError> {
         counters::ADD_TX_INITIATED.inc();
+        let session_id = ClientSession::from(msg.channel_id.clone());
         let tx_context = self.enclave.client_tx_propose(msg.into())?;
 
         // Cache the transaction. This performs the well-formedness checks.
         let tx_hash = self.tx_manager.insert(tx_context).map_err(|err| {
             if let TxManagerError::TransactionValidation(cause) = &err {
                 counters::TX_VALIDATION_ERROR_COUNTER.inc(&format!("{cause:?}"));
+
+                let tracking_window = Duration::from_secs(5); // TODO, placeholder before draft PR gets published.
```

Comment:
nit: remove todo

Reply:
Addressed in [6b3861f](https://github.com/mobilecoinfoundation/mobilecoin/pull/3208/commits/6b3861fa7beed20555822ea943a9552d3c9d035b)